package.json

```diff
@@ -2,7 +2,7 @@
   "author": "Transcend Inc.",
   "name": "@transcend-io/cli",
   "description": "Small package containing useful typescript utilities.",
-  "version": "6.14.3",
+  "version": "6.14.4",
```

Comment:
Or should this be `6.15.0`? @anotherminh

Reply:
See my other comment!

Reply:
What other comment? I see the one about how to run it (which worked, by the way) but not about whether this would be a minor or a dot change

Reply:
@anotherminh did you have a comment about version? (Realized you might not have seen my last message since I forgot to tag)

Reply:
Oh hmmm sorry i was confused about this question! My b!

I think minor sounds good

Reply:
minor it is. Thanks!